client/src/pages/Login.js

```diff
@@ -1,7 +1,106 @@
+                <hr size="0.5" width="3%" color={green['A400']}></hr>
+                <Box className={classes.formHeaderBody}>
+                    <span>New here? </span>
+                    <Link className={classes.formHeaderBodyLink} to='signup'>Sign Up</Link>
+                </Box>
+            </Grid>
+            <Grid container item xs={12} alignItems="center" direction='column'>
+                <form onSubmit={handleSubmit(onSubmit)} className={classes.formWidth}>
+                    <Grid container item xs={12} alignItems="center" direction='column' spacing={3}>
+                        <Grid container item xs={12} md={4}>
+                            <TextField
+                                fullWidth
+                                autoComplete='Off'
+                                label="Email address"
+                                variant="outlined"
+                                name='email'
+                                {...register('email', {
+                                    required: 'email is required',
+                                    pattern: {
+                                        value: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i,
+                                        message: "Please enter a valid email address"
+                                    }
+                                })}
+                                error={Boolean(errors.email)}
+                                helperText={errors.email ? errors.email.message : null}
+                            />
+                        </Grid>
+                        <Grid container item xs={12} md={4}>
+                            <TextField
+                                fullWidth
+                                type='password'
+                                label="Password"
+                                variant="outlined"
+                                name='password'
+                                {...register('password', {
+                                    required: 'password is required',
+                                    minLength: { value: 6, message: 'password must be at least 6 characters'} })
+                                }
+                                error={Boolean(errors.password)}
+                                helperText={errors.password ? errors.password.message : null}
+                            />
+                        </Grid>
+                        <Grid container item xs={12} md={4}>
+                            <FormGroup row>
+                                <FormControlLabel
+                                    control={
+                                    <Checkbox
+                                        name='remember'
+                                        {...register('remember')}
+                                        // onChange={e => setRememberMe(e.target.checked)}
+                                        // value={defaultValues.remember}
```

Comment:
It's generally best to remove commented out code before merging your branch so there isn't any in dev

Reply:
for sure. thanks